pulsar-metadata/src/main/java/org/apache/pulsar/metadata/bookkeeper/AbstractMetadataDriver.java

```diff
@@ -98,7 +98,9 @@ void createMetadataStore() throws MetadataException {
 
             String url;
             try {
-                url = conf.getMetadataServiceUri().replaceFirst(METADATA_STORE_SCHEME + ":", "");
+                url = conf.getMetadataServiceUri()
+                        .replaceFirst(METADATA_STORE_SCHEME + ":", "")
+                        .replace(";", ",");
```

Comment:
Do we still need to replace it here? Since we removed the replacement before.

Reply:
If you set `metadataServiceUri` in bookkeeper.conf with semicolon, like zk1:2181;zk2:2181/test, without replace semicolon with comma. metadataStore will be failed created.

Reply:
@codelipenghui

Reply:
> Set metadataServiceUri in bookkeeper.conf with comma, it will also cause this issue, Because of BK side

This is a known issue https://github.com/apache/bookkeeper/issues/2900